src/Framework/FieldsAPI/Checkbox.php

```diff
@@ -2,48 +2,88 @@
 
 namespace Give\Framework\FieldsAPI;
 
+use Give\Framework\Exceptions\Primitives\RuntimeException;
+
 /**
- * @since 2.12.0
+ * A single checkbox input. If supported, an indeterminate value is represented as a null value, as opposed to true or
+ * false value.
```

Comment:
Do we still need to mention "indeterminate" here?

Reply:
Hah! Nope, good catch. Resolved in 6219e4831